From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@ShawnJackson__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-shawja). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

carolinacmoravia committed:
CC4446: Endpoint typo

@scottwhi, 
Could you check if the proposed file change is correct, please? Mamy thanks in advance.

v-viinde committed:
USER STORY 1040392 SaaS App Tutorial: Perception United States (Non-UltiPro)

v-viinde committed:
Updated

v-viinde committed:
Updated

v-viinde committed:
Updated

CFreemanwa committed:
adding enable-profiler-cloud files

CFreemanwa committed:
fixing json links

Santoshtambe committed:
BUG_1046112 "REST API" link updated

Santoshtambe committed:
1046112 - file validated

CFreemanwa committed:
fixing the TOC and responding to Acrolynx

CFreemanwa committed:
misc updates for walkthrough

CFreemanwa committed:
fixing a link

CFreemanwa committed:
fixing another link

CFreemanwa committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into july27master

CFreemanwa committed:
responding to PM feedback

Commit:
Adding new articles for Redis and Cosmos DB

Commit:
Updating Cosmos DB article based on Acrolinx feedback

Commit:
Updating images and some small grammar changes

Commit:
Updating two images

Commit:
Updated azure-document-db-spring-boot-starter version

Commit:
Updated corresponding image

YuriDio committed:
adding article

YuriDio committed:
removing redirect

Commit:
Update iot-hub-raspberry-pi-web-simulator-get-started.md

Commit:
Update iot-hub-raspberry-pi-web-simulator-get-started.experimental.md

cherylmc committed:
update format

cherylmc committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into format4

YuriDio committed:
removing redirect 2

CFreemanwa committed:
changing reference link for analytics to new site

cherylmc committed:
update

cherylmc committed:
update

cherylmc committed:
update

cjgronlund committed:
freshness pass

CFreemanwa committed:
responding to PR review comments

cjgronlund committed:
higher value search term - example

cjgronlund committed:
fixes

cjgronlund committed:
kw

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-shils committed:
Merge pull request #18307 from cjgronlund/fresh-july

freshness pass - Azure ML sample experiments

Ja-Dunn committed:
Merge pull request #18219 from CFreemanwa/july27master

adding enable-profiler-cloud files

Ja-Dunn committed:
Merge pull request #18288 from YuriDio/JulyUpdates_10

adding article

Ja-Dunn committed:
Merge pull request #18241 from Santoshtambe/BUG_1046112

BUG_1046112 "REST API" link updated

Ja-Dunn committed:
Merge pull request #17741 from v-viinde/perceptionunitedstates

USER STORY 1040392 SaaS App Tutorial: Perception United States (Non-U…

v-shils committed:
Merge pull request #18296 from cherylmc/format4

update format + include

Ja-Dunn committed:
Merge pull request #18293 from shizn/patch-20

iot hub web simulator tutorial experiment finished

tynevi committed:
Merge pull request #15781 from carolinacmoravia/patch-251

CC4446: Endpoint typo

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into 20170721

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPublicRepo

Commit:
Updating articles based on editor feedback

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

v-alje committed:
Merge pull request #18321 from MicrosoftDocs/FromPublicRepo

From public repo

Ja-Dunn committed:
Merge pull request #18275 from rmcmurray/20170721

Adding new articles for Spring Boot with Redis and Cosmos DB
